docs/getting_started/install_manually.md

```diff
@@ -212,290 +212,116 @@ Open your project in the browser and you should see the welcoming page.
+2. You can accept the default options for `database_driver`, `database_host` and `database_port`
+3. For `database_name` and `database_user`, replace them if you customized those values during configuration.
+4. If you set a password for your database user, enter it when prompted for `database_password`.
 
-h. Change directory permissions:
+The installer should continue once you've completed the manual portion of the installation process.
 
-``` bash
-rm -rf var/cache/* var/logs/* var/sessions/*
-sudo chmod +a "_www allow delete,write,append,file_inherit,directory_inherit" var web/var
-sudo chmod +a "`whoami` allow delete,write,append,file_inherit,directory_inherit" var web/var
-```
+#### Install eZ Platform
 
-i. Install eZ Platform:
+Create `clean` installation in development environment and a database with:
 
 ``` bash
-php bin/console ezplatform:install ezplatform-clean
+php -d memory_limit=-1 composer.phar ezplatform-install
 ```
 
-You will be able to see your page under <http://ez1.lh> (or the address you chose in preparation). Please note that a clean install of eZ Platform doesn’t include DemoBundle anymore.
+If Composer asks for your token, you must log in to your GitHub account and generate a new token
+(edit your profile, go to Developer settings > Personal access tokens and Generate new token with default settings).
+This operation is performed only once when you install eZ Platform Enterprise Edition for the first time.
 
 !!! enterprise
 
     ###### Enable Date-based Publisher
 
     To enable delayed publishing of Content using the Date-based Publisher, see [above](#enable-date-based-publisher)
 
-### 10. Optional
-
-a. Install PHP 5.6 with opcache extension:
-
-``` bash
-brew install -v homebrew/php/php56
-chmod -R ug+w $(brew --prefix php56)/lib/php
-brew install -v php56-opcache
-```
-
-b. Add proper `date.timezone` settings:
-
-``` bash
-sudo vi /usr/local/etc/php/5.6/php.ini
-```
-
-c. Uncomment and modify:
-
-```
-date.timezone = "Europe/Warsaw"
-(…)
-Increase memory_limit value for eZ Platform:
-memory_limit = 4G
-(…)
-```
-
-d. Disable errors showing:
+!!! tip
 
-``` bash
-display_errors = Off
-```
+    You can use PHP's built-in server after installation: `php bin/console server:start`.
 
-e. Change default PHP parser used by Apache:
+    If you want to use an Apache web server, you need to prepare a Virtual Host and [set up directory permissions](set_up_directory_permissions.md)
 
-``` bash:
-sudo vi /private/etc/apache2/httpd.conf
-```
+### Set up Virtual Host
 
-f. Find and comment the following line:
+To set up Virtual Host, you can use the template provided with eZ Platform:
 
```

Comment:
Linux installation guide is more detailed. If we don't want to describe equally macOS installation, we can just mention that the file `<your_site_name>.conf` should be edited, because we give below only a copy instruction. After that server restart is needed, so it could be added too.